README.md

```diff
@@ -1,8 +1,12 @@
-[![Documentation Status](https://readthedocs.org/projects/rocketpyalpha/badge/?version=latest)](https://rocketpyalpha.readthedocs.io/en/latest/?badge=latest)
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/giovaniceotto/rocketpy/blob/master/docs/notebooks/getting_started_colab.ipynb)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/giovaniceotto/RocketPy/master?filepath=docs%2Fnotebooks%2Fgetting_started.ipynb)
-[![Downloads](https://pepy.tech/badge/rocketpyalpha)](https://pepy.tech/project/rocketpyalpha)
+[![PyPI](https://img.shields.io/pypi/v/rocketpy?color=g)](https://pypi.org/project/rocketpy/)
+[![Downloads](https://pepy.tech/badge/rocketpyalpha)](https://pepy.tech/project/rocketpy)
+[![Documentation Status](https://readthedocs.org/projects/rocketpyalpha/badge/?version=latest)](https://rocketpyalpha.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Is it possible to use "rocketpy" instead of rocketpyalpha here?

Reply:
For Downloads, yes! Thanks for catching my mistake. I added a commit to fix this.

For Documentation, unfortunately we cannot change it to `rocketpy`.

Reply:
Mmmm that's weird, @giovaniceotto . I wonder why we don't ask the docs ownership for us, just like we did with pyPI

Reply:
We can ask as well. I'll look into it and check if it is worth it. There are plans to add a custom domain such as `rocketpy.org/docs` instead of using the ReadTheDocs one.

Anyways, I believe that this can be changed in a future Pull Request.

Reply:
OK